library/ssl_tls13_client.c

```diff
@@ -890,16 +1052,22 @@ static int ssl_tls13_cipher_suite_is_offered( mbedtls_ssl_context *ssl,
  *    Extension extensions<6..2^16-1>;
  * } ServerHello;
  */
+#define ALERT_UNSUPPORTED_EXTENSION_FOUND   ( 1 << 0 )
+#define ALERT_ILLEGAL_PARAMETER_FOUND       ( 1 << 1 )
```

Comment:
It is not really possible to have both alerts set at the same time thus I would go just with the code of the detected alert (see below).